src/OowpQuery.php

```diff
@@ -45,27 +45,27 @@ public function getIterator()
         return new \ArrayIterator($this->posts);
     }
 
-    public function offsetExists($offset)
+    public function offsetExists($offset): bool
     {
         return isset($this->posts[$offset]);
     }
 
-    public function offsetGet($offset)
+    public function offsetGet($offset): bool
```

Comment:
this is the wrong return type. should be `mixed`
https://www.php.net/manual/en/class.arrayaccess.php